store/modules/actions/payment.js

```diff
@@ -80,6 +81,7 @@ export async function calculateISCNTxTotalFee({ commit },
     url,
   } = payload;
   const { ISCNTotalFee } = await calculateISCNTotalFee({
+    from,
```

Comment:
We should use `cosmosWallet` (or `from` if that is better )for all input for consistency, but not both